src/features/content-answers/__tests__/InlineError.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { render, screen } from '@testing-library/react';
+
+import InlineError from '../InlineError';
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { ElementsXhrError } from '../../../common/types/api';
+
+import messages from '../messages';
+
+jest.mock('react-intl', () => jest.requireActual('react-intl'));
+
+describe('features/content-answers/InlineError', () => {
+    const IntlWrapper = ({ children }: { children?: React.ReactNode }) => {
+        return <IntlProvider locale="en">{children}</IntlProvider>;
+    };
```

Comment:
we dont need the intl wrapper here since youre not apply it directly to this component 
```suggestion

```

Reply:
this is to handle the console.err `[React Intl] Could not find required `intl` object. <IntlProvider> needs to exist in the component ancestry. Using default message as fallback.`

Reply:
there is a global mock for FormattedMessage shown below https://github.com/box/box-ui-elements/blob/4f6c21941c2f0d05a830ffd2c3dfdca4b946895c/src/__mocks__/react-intl.js#L16-L17

In order to test the text message, we need to mock it again and use the actual react-intl which requires IntlWrapper.